src/docs/goose-book/src/getting-started/running.md

```diff
@@ -4,87 +4,42 @@ We will use Cargo to run our example load test application. It's best to get in
 
 ```bash
 $ cargo run --release
-   Compiling loadtest v0.1.0 (/home/jandrews/devel/rust/loadtest)
-    Finished dev [optimized] target(s) in 3.56s
+    Finished release [optimized] target(s) in 0.23s
      Running `target/release/loadtest`
 Error: InvalidOption { option: "--host", value: "", detail: "A host must be defined via the --host option, the GooseAttack.set_default() function, or the GooseTaskSet.set_host() function (no host defined for LoadtestTasks)." }
 ```
 
-The load test fails with an error as it hasn't been told the host you want to load test. So, let's try again, this time passing in the `--host` flag. After running for a few seconds, press `ctrl-c` to stop the load test:
+The load test fails with an error as it hasn't been told the host you want to load test. So, let's try again, this time passing in the `--host` flag. We will also add `--report-file` flag, which will generate a a HTML report. Same information will also be printed to the command line (without graphs). After running for a few seconds, press `ctrl-c` to stop the load test:
```

Comment:
`a a HTML report` should be `a HTML report`

Reply:
also, let's link to `metrics.html` in this same section for an example of the metrics output if `--report-file` isn't used.

(Also, not really relevant, but I discovered today that there's some information in the CLI metrics that aren't in the HTML report metrics: I'll file a bug when I get a chance)

Reply:
I noticed the same thing as well - median is missing in HTML.

Reply:
Done.